contrib/gdk-pixbuf/.clang-format

```diff
@@ -0,0 +1,21 @@
+Language: Cpp
```

Comment:
Add a comment to note the source of this coding style. I think we can use the commit message as a comment.

Optional: Mention that you reduced IndentWidth from 8 to 4.